.travis.invenio.cfg

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2015 CERN.
+#
+# Zenodo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Travis-CI configuration."""
+import getpass
+
+CFG_BIBSCHED_PROCESS_USER = getpass.getuser()
```

Comment:
Looking into [travis build](https://travis-ci.org/inveniosoftware/invenio-groups/jobs/62393660) I think `CFG_BIBSCHED_PROCESS_USER` is needed:
```
ERROR: You must run "webcoll" as the same user that runs your Apache server
process (seems to be "www-data").
You may want to do "sudo -u www-data webcoll ..." to do so.
If you think this is not right, please set CFG_BIBSCHED_PROCESS_USER
appropriately and rerun "inveniocfg --update-config-py".
```